README.md

```diff
@@ -149,7 +151,7 @@ and max number of retries
     * [.getEventsFromJournal(journalUrl, [eventsJournalOptions], [fetchResponseHeaders])](#EventsCoreAPI+getEventsFromJournal) ⇒ <code>Promise.&lt;object&gt;</code>
     * [.getEventsObservableFromJournal(journalUrl, [eventsJournalOptions], [eventsJournalPollingOptions])](#EventsCoreAPI+getEventsObservableFromJournal) ⇒ <code>Observable</code>
     * [.verifySignatureForEvent(event, clientSecret, signatureHeaderValue)](#EventsCoreAPI+verifySignatureForEvent) ⇒ <code>boolean</code>
```

Comment:
Shall we say via readme also that this method is deprecated?

Reply:
So sorry Abhishek, but right now it looks really confusing (see below screenshots) the way we have marked the methods as deprecated in `README`. Can we simply add a `[deprecated]` in front of the functions?

> ![Screenshot 2022-01-19 at 3 23 35 PM](https://user-images.githubusercontent.com/6910192/150107059-29c67437-d54d-4644-9729-4e4f7f358eb8.png)

>![Screenshot 2022-01-19 at 3 23 53 PM](https://user-images.githubusercontent.com/6910192/150107071-c54f4dfe-57df-4944-9100-6c2ef008a70b.png)

Reply:
Or we can do something like this:

1. https://github.com/adobe/aio-lib-campaign-standard/blob/516eea655b3fb297c7adc903d74aedb2e999707e/README.md
2. https://github.com/adobe/aio-lib-runtime/blob/f9f55bbd45e90b08cab693d4a60fb7d4a1428ba8/README.md